UEBPrep/config/unicorn.rb

```diff
@@ -1,5 +1,11 @@
 worker_processes Integer(ENV["WEB_CONCURRENCY"] || 3)
-timeout 15
+
+if ENV['RAILS_ENV']=='development'
+  timeout 150
```

Comment:
15 seconds isn't long enough for debugging purposes. Otherwise our server will terminate your connection if any request takes longer than 15 seconds to run-such as when you have a debugging statement in a controller and are poking around.